usagov/usagov-2021
# USAGOV-1236: Check for Spanish nav on English home page in Tome output

Cancels publish, removes English home page, and sets the flag to re-run Tome regardless of content changes


## Jira Task

https://cm-jira.usa.gov/browse/USAGOV-1236

## Description

Adds yet another clause to tome-sync.sh, this time checking for a word we expect in the Spanish mobile nav, that we don't expect on the English home page. (In this case, I chose the value of the aria-label for the nav.) This mirrors the "English nav on Spanish home page" case.

I've checked "Bugfix" but this is really more of a workaround.

## Type of Changes

  - Add screenshot showing what it should look like

## Testing Instructions



### Requires New Config

### Requires New Content

### Validation Steps
The problem we're checking for here is intermittent, so running tome to try to prove it doesn't make sense, although if I've broken publishing altogether we will see that when this is deployed to dev! 

Please validate my "grep" statement against the English and Spanish home pages and just review the changes for sensibility.

## Security Review




## Reviewer Reminders
- Reviewed code changes
- Reviewed functionality
- Security review complete or not required

## Post PR Approval Instructions
Follow these steps as soon as you merge the new changes.

1. Go to the [USAGov Circle CI project](https://app.circleci.com/pipelines/github/usagov/usagov-2021).
2. Find the commit of this pull request.
3. Build and deploy the changes.
4. Update the Jira ticket by changing the ticket status to `Review in Test` and add a comment. State whether the change is already visible on [cms-dev.usa.gov](http://cms-dev.usa.gov/) and [beta-dev.usa.gov](http://beta-dev.usa.gov/), or if the deployment is still in process.
